### DISCUSSION TITLE
Add option to set different reps to every set

### DISCUSSION THREAD

**Discussion by spar34vi** [LiamMorrow/LiftLog#836] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have some exercises in my program that have a different number of reps for every set, e.g. Leg Press First set 20 reps, second set 16 reps, third set 12 reps, fourth set 8 reps.

**Describe the solution you'd like**
An option to set different reps to every set.

**Describe alternatives you've considered**
Writing the number of reps in the notes.

**Additional context**
.

**Comment by LiamMorrow** [comment]
This is now in release 4.18.1 🎉
